src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const currentTabId = tabs.some(tab => tab.id === activeTabId)
+    ? activeTabId
+    : tabs[0]?.id;
```

Comment:
The logic for determining `currentTabId` is correct, but consider using optional chaining consistently for robustness. The line `: tabs[0]?.id;` uses optional chaining, which is good, but ensure that `tabs` is always defined before accessing `tabs[0]`.